nltk/parse/transitionparser.py

```diff
@@ -752,6 +752,7 @@ def demo():
     >>> parser_std.train([gold_sent],'temp.arcstd.model', verbose=False)
      Number of training examples : 1
      Number of valid (projective) examples : 1
+    >>> input_file.close()
```

Comment:
Temporary files were being removed before they were closed, causing exceptions (and leftover files on my device...)